astropy/coordinates/tests/test_arrays.py

```diff
@@ -100,29 +100,28 @@ def test_array_coordinates_creation():
     """
     Test creating coordinates from arrays.
     """
-    from .. import Angle
-    from .. import ICRS, CartesianRepresentation
+    from .. import Angle, SkyCoord, CartesianRepresentation
 
-    c = ICRS(np.array([1, 2])*u.deg, np.array([3, 4])*u.deg)
+    c = SkyCoord(np.array([1, 2])*u.deg, np.array([3, 4])*u.deg)
     assert not c.ra.isscalar
```

Comment:
@taldcroft Yeah, it could be that this is already covered.  I tend towards possibly testing something twice over possibly not testing it at all to be on the safe side...